packages/powershell.rb

```diff
@@ -3,15 +3,18 @@
 class Powershell < Package
   description 'Powershell is a cross-platform, task-based command-line shell and scripting language that helps rapidly automate tasks that manage operating systems and processes'
   homepage 'https://docs.microsoft.com/en-us/powershell/'
-  version '7.0.2'
+  version '7.0.3'
   compatibility 'aarch64,armv7l,x86_64'
   case ARCH
-  when 'aarch64', 'armv7l'
-    source_url 'https://github.com/PowerShell/PowerShell/releases/download/v7.0.2/powershell-7.0.2-linux-arm32.tar.gz'
-    source_sha256 '3f884c8eb97cbf28402d5ee5c64c50c556c87779f08a7eb01062c5b7567f9246'
+  when 'armv7l'
+    source_url 'https://github.com/PowerShell/PowerShell/releases/download/v7.0.3/powershell-7.0.3-linux-arm32.tar.gz'
+    source_sha256 'fe6da9f90fd5026c4ab446a31f565144342a8d507a73d428305bc36c1ad92644'
+  when 'aarch64'
+    source_url 'https://github.com/PowerShell/PowerShell/releases/download/v7.0.3/powershell-7.0.3-linux-arm64.tar.gz'
+    source_sha256 'b5b6db3e5605a6a7675809cd49573f352e1eb6ca9d65890edda1b3f27a8ada56'
```

Comment:
This will not install on aarch64 since it still uses a 32-bit userland.  Please combine with `armv7l` like the earlier version.

Reply:
Done